src/Makefile.leveldb.include

```diff
@@ -11,6 +11,7 @@ EXTRA_LIBRARIES += $(LIBMEMENV_INT)
 LIBLEVELDB += $(LIBLEVELDB_INT) $(LIBCRC32C)
 LIBMEMENV += $(LIBMEMENV_INT)
 
+LEVELDB_CPPFLAGS += -DFALLTHROUGH_INTENDED=[[fallthrough]]
```

Comment:
This should be in `LEVELDB_CPPFLAGS_INT` and down with the other defines.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/21430#issuecomment-873826524).